fourmolu.yaml

```diff
@@ -4,13 +4,14 @@ import-export-style: leading
 fixities:
   - infixr 9  .
   - infixr 5  ++
-  - infixl 4  <$, <$>
+  - infixr 4   <$, <$>
   - infixl 2  >>, >>=
   - infixr 2  =<<, <=<
   - infixr 1  $, $!
   - infixl 4 <*>, <*, *>, <**>
   - infixl 6 ^.
   - infixl 3 =.
+  - infixl 2 <|>
```

Comment:
Out of curiosity, I started looking into the fixities of operators in `base`. I get different numbers for some of them. 😳

```
module GHC.Base
infixr 9  .
infixr 5  ++
infixl 4  <$
infixl 1  >>, >>=
infixr 1  =<<
infixr 0  $, $!

infixl 4 <*>, <*, *>, <**>

infixr 6 <>
infixl 3 <|>
infixr 5 :|
```

The GHC.Base module matches up with the [Haskell2010 standard][2010].

  [2010]: https://www.haskell.org/onlinereport/haskell2010/haskellch4.html#x10-820061

Reply:
I will try them out of curiosity :-)

Reply:
the meaning is different but I cannot explain it, I will keep the numbers but fix `l` and 'r' to match our style